rack-test.gemspec

```diff
@@ -26,6 +26,7 @@ request helpers feature.
   s.add_development_dependency 'rspec', '~> 3.6'
   s.add_development_dependency 'sinatra', '>= 1.0', '< 3'
   s.add_development_dependency 'rdoc', '~> 5.1'
+  s.add_development_dependency 'rubocop', '>= 0.49', '< 0.50'
```

Comment:
This line is to prevent warning from `gem build *.gemspec`.

`s.add_development_dependency 'rubocop', '~> 0.49.1'`displays warning message "too strict for development dependency.".
But I want to pin the version with '0.49.x'. Because rubocop changes/adds their rules often.